src/client/game.cpp

```diff
@@ -535,7 +537,10 @@ class GameGlobalShaderConstantSetter : public IShaderConstantSetter
 		if (m_fog_enabled && !*m_force_fog_off)
 			fog_distance = *m_fog_range;
```

Comment:
Yes, because it does not increase the range of what the client can see.
(as an aside I do not understand the need to disable fog at all. This is different from unlimited viewing range.)

Reply:
(If fog is on, the user can just see stuff inside a sphere, with fog off, they can see whole mapblocks, so it's beneficial.)
We should maybe restrict disabling fog to the debug flag (later PR).

Reply:
The user can still just see a sphere, the sphere of mapblocks in range, at worst a partial block edge is visible.
I like limiting fog-off to debug!